src/upload.ts

```diff
@@ -92,6 +97,7 @@ async function assumeRoles(
         RoleArn: role,
         RoleSessionName: `mevblocker-dune-sync-${timestamp}`,
         ExternalId,
+        DurationSeconds: 3600,
```

Comment:
> Requests longer STS sessions to lower ExpiredToken frequency.

According to the [docs](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html) 3600 is the default.